concepts/design-tokens/README.md

```diff
@@ -0,0 +1,234 @@
+# Design tokens
+
+> Design tokens are an agnostic way to store variables such as typography,
+> color, and spacing so that your design system can be shared across platforms
+> like iOS, Android, and regular ol' websites.
+
+-- <cite>Robin Rendle "[What are design tokens]", CSS-Tricks</cite>
+
+## What we want to achive with Design tokens
+
+In the current state of the Barista components library, we do share some basic
+color, typography and spacing definitions through scss mixins and variables
+within the angular ecosystem. Which works very well for color definitions, did
+not prove to be consistent for spacings and typography definitions. As our plans
+for Barista do not only target Angular, but a more framework agnostic component
+library, we are re-evaluating the way on how we deal with design tokens.
+
+Extracted design tokens should enable us to be extremely consistent within the
+system. With everything, that is relevant to the design-language baked into
+tokens, we should be able to enforce usage of these tokens with linting rules
+like [stylelint declaration strict value].
+
+## Naming of design tokens
+
+To prevent conflicts with css custom properties or other variables in output
+files, we should prefix all our provided design tokens, mixins or exported
+constants with a `dt-`/`dt` prefix.
+
+## Structure
+
+Tokens are usually stored in some sort of structual data (yml, json,
+database,...). Which format we are ultimately choosing is still up for
+discussion.
+
+Just like the format, the file structure is also up for discussion. There are
+multiple ways of structuring the design tokens into a folder structure, and a
+conversion tool like [theo] will let you customize
+[input and output formats quite easily](#output-and-formats).
+
+Here are some variations of design tokens structure in the wild:
+
+- [Salesforce design tokens]
```

Comment:
should this be a link?

Reply:
Yes, and it is. This is a shorthand link syntax in markdown. 
Link targets are collected at the bottom of the md file: 
https://github.com/dynatrace-oss/barista/pull/633/files#diff-3f1f6d3a5d17588e9f3aa79c5b52262eR228